src/test/java/domain/LottoRandomTest.java

```diff
@@ -0,0 +1,22 @@
+package domain;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LottoRandomTest {
+    @Test
+    void 로또_난수_범위_테스트() {
+        // given
+        LottoRandom lottoRandom = new LottoRandom();
+        List<Integer> lottoNum=lottoRandom.getLottoRandomNum();
+        // when&then
+        for (int num : lottoNum) {
+            assertTrue(1 <= num && num <= 45,"범위오버");
+        }
+    }
```

Comment:
```suggestion
        // given
        LottoRandom lottoRandom = new LottoRandom();
        
        // when 
        List<Integer> lottoNum=lottoRandom.getLottoRandomNum();
        
        // then
        for (int num : lottoNum) {
            assertTrue(1 <= num && num <= 45,"범위오버");
        }
    }
```

1. 가독성을 위해서 이렇게 바꾸면 더 좋을 것 같아요!
2. 정렬이 안되어 있습니다
3. 테스트가 잘못된 것 같습니다. `범위오버`라는 메시지를 바꾸면 테스트가 실패해야할 것 같은데 실패하지 않습니다! 다른 메서드를 활용해보면 어떨까요?